internal/k8shandler/reconciler.go

```diff
@@ -42,11 +49,21 @@ func Reconcile(requestCluster *logging.ClusterLogging, requestClient client.Clie
 	if clusterLoggingRequest.IncludesManagedStorage() {
 		// Reconcile Log Store
 		if err = clusterLoggingRequest.CreateOrUpdateLogStore(); err != nil {
+			telemetry.Data.CLInfo.Set("healthStatus", "1")
+			erru := telemetry.UpdateCLMetrics()
+			if erru != nil {
+				log.V(1).Error(erru, "Error in updating clo metrics for telemetry")
+			}
 			return fmt.Errorf("unable to create or update logstore for %q: %v", clusterLoggingRequest.Cluster.Name, err)
 		}
 
 		// Reconcile Visualization
 		if err = clusterLoggingRequest.CreateOrUpdateVisualization(); err != nil {
+			telemetry.Data.CLInfo.Set("healthStatus", "1")
+			erru := telemetry.UpdateCLMetrics()
+			if erru != nil {
+				log.V(1).Error(erru, "Error in updating clo metrics for telemetry")
+			}
```

Comment:
This pattern recurs many times. If failure to update telemetry is not a real error then perhaps encapsulate this in 
```
func telementry.UpdateCLMetricsNoErr() {
erru := telemetry.UpdateCLMetrics()
			if erru != nil {
				log.V(1).Error(erru, "Error in updating clo metrics for telemetry")
			}
}
```